agenta-backend/agenta_backend/main.py

```diff
@@ -90,7 +92,9 @@ async def lifespan(application: FastAPI, cache=True):
 
 if os.environ["FEATURE_FLAG"] in ["cloud", "ee", "demo"]:
     import agenta_backend.ee.main as ee
+    from agenta_backend.ee.routers import organization_router
 
+    app.include_router(organization_router.router, prefix="/organizations")
```

Comment:
not sure we need this here.. shouldn't be in ee ?

Reply:
moved to ee.